content/en/quickstart/device-to-device.md

```diff
@@ -11,3 +11,243 @@ menu:
     weight: 20
 toc: true
 ---
+This guide will walk you through the process of discovering/on-boarding/controlling your secure [OCF Device](https://openconnectivity.org/specs/OCF_Device_Specification_v2.2.3.pdf) using **plgd/sdk** library on your PC.
+
+## Prerequisite
+- The Cloud Server example should be running on your Raspberry Pi at the step of [Create device](../create-device)
```

Comment:
The Cloud Server [runs](../create-device) on your Raspberry Pi

Reply:
> Still miss this introduction to how does it work. I am pretty sure it's in the core spec.

What do you mean by above sentence? Do you want to add more description at the start of `This guide will walk...` about device core spec ?

Reply:
No, just exchange your sentence with mine. It's shorter, simpler and saying the same :)